tasks/lula.yaml

```diff
@@ -0,0 +1,78 @@
+variables:
+  - name: OSCAL_FILE
+
+tasks:
+  - name: lula-lint
+    description: Validating OSCAL against Schema
+    actions:
+      - description: Runs Lula Lint command
+      - cmd: |
+          OSCAL_FILES_LIST=$(echo "$OSCAL_FILE")
+
+          lint_errors=0
+
+          # Loop through each file and lint
+          for target in $OSCAL_FILES_LIST; do
+              # Check if the file exists
+              if [ ! -f "$target" ]; then
+                  echo "File $target does not exist"
+                  lint_errors=$(($lint_errors + 1))
+                  continue
+              fi
+
+              # Lint the file with Lula
+              echo "Linting $target"
+              lula tools lint -f "$target"
+              if [ $? -ne 0 ]; then
+                  echo "Linting failed for $target"
+                  lint_errors=$(($lint_errors + 1))
+              fi
+          done
+
+          # Check if there were any lint errors
+          if [ "$lint_errors" -ne 0 ]; then
+              echo "Linting failed for one or more files."
+              exit 1
+          else
+              echo "All files linted successfully."
+          fi
```

Comment:
Is there an issue in Lula for this looping logic? If this is how it is expected to be used then it should probably be a feature / flag in Lula itself.

Reply:
This is in the works for Lula pending a [go-oscal PR](https://github.com/defenseunicorns/go-oscal/pull/261), then a [Lula PR](https://github.com/defenseunicorns/lula/pull/481). 

Haven't ran into the need to actually lint more than 1 oscal file at a time but will eventually.